config/environments/production.rb

```diff
@@ -21,7 +21,7 @@
   # config.require_master_key = true
 
   # Disable serving static files from `public/`, relying on NGINX/Apache to do so instead.
-  config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
+  # config.public_file_server.enabled = false
```

Comment:
How is this changing the app?

Reply:
The default is true.

The ENV is set on Heroku. 

The result; the same.